### PR TITLE
Add permission to release gitlab-branch-source-plugin

### PR DESCRIPTION
**Pull request by baymac**
# Description

# Submitter checklist for changing permissions

Plugin Git Repository: https://github.com/jenkinsci/gitlab-branch-source-plugin/
HOSTING issue: https://issues.jenkins-ci.org/browse/HOSTING-795
Wiki page: https://wiki.jenkins.io/display/JENKINS/GitLab+Branch+Source+Plugin

CC: @markyjackson-taulia @casz @justinharringa @LinuxSuRen 

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

#### Merge permission to GitHub repository
- [X] Check this if newly added person also needs to be given merge permission to the GitHub repo.

### CODE REVIEW

**Comment by batmat** [comment]
`User name not known to Artifactory: LinuxSuRen`...

**Comment by batmat** [comment]
And indeed, https://issues.jenkins-ci.org/secure/ViewProfile.jspa?name=LinuxSuRen yields a "user not found".

Please follow the "Details" link above to check this yourself to make this ready. Thanks

**Comment by batmat** [comment]
Also, please make sure to check the boxes after actually checking it :). Unchecked for now, I believe this is the problem:
image

**Comment by baymac** [comment]
@batmat I thought I ensured that I was using the LDAP users. Since in a different [pr](https://github.com/jenkins-infra/repository-permissions-updater/pull/1218/files#diff-c92fafaedc664e3cfd08c3cb271fdbefR11) `LinuxSuRen` was a valid user. I made an assumption although my bad I didn't reverify.

**Comment by baymac** [comment]
@batmat thanks for the info. My bad.

File: permissions/plugin-gitlab-branch-source.yml [hunk 1]
```diff
@@ -8,4 +8,4 @@ developers:
 - "jequals5"
 - "justinharringa"
 - "casz"
-- "LinuxSuRen"
+- "surenpi"
```
Comment (LinuxSuRen) [inline permissions/plugin-gitlab-branch-source.yml]: sorry, I missed this part

**Comment by batmat** [comment]
Not merging it immediately, because otp right now and if I merge I'll forget to also provide merge permission as requested.

@baymac well grant merge permission to everyone allowed to release right?

**Comment by baymac** [comment]
@batmat thanks for asking. We all worked together to build this plugin, so it's fine. Please give merge permission to all.

**Comment by batmat** [comment]
Actually, given this is a followup of https://issues.jenkins-ci.org/browse/HOSTING-795, everybody should already have merge permission since @slide ran the hosting command.

If not, please reach out here or in https://issues.jenkins-ci.org/browse/HOSTING-795.

Welcome on board, and thanks again for your great work!

**Comment by baymac** [comment]
Thanks a lot @batmat.

**Pull request merged**